src/components/Tabs.vue

```diff
@@ -101,7 +103,7 @@
     data() {
       return {
         active: this.readme ? 'readme' : 'console',
-        tabHeight: 280,
+        tabHeight: defaultBottomPanelHeight,
```

Comment:
Right... Well, I thought about that, but I'd almost rather not spam vuex with state updates. The way it is now, `tabHeight` is used to track the height and render it, and bottomPanelHeight is only set _after_ the movement is over. Basically there's only one vuex mutation instead of many.